lib/smart_answer/calculators/pay_leave_for_parents_calculator.rb

```diff
@@ -11,6 +11,7 @@ class PayLeaveForParentsCalculator
     attr_accessor :mother_earned_more_than_lower_earnings_limit
     attr_accessor :mother_worked_at_least_26_weeks
     attr_accessor :mother_earned_at_least_390
+    attr_accessor :salary_1_66_weeks
```

Comment:
Is this used anywhere? It seems odd that no other code in the flow or template has had to change.